data/recommendation/notify.js

```diff
@@ -131,29 +130,25 @@ class Notify {
   showWhyBox() {
     this.whyBoxShowing = true;
     const lastRec = document.querySelector('.addon-box:last-child');
-    lastRec.className = 'hiddenRec';
-    lastRec.setAttribute('hidden', 'true');
+    lastRec.classList.add('hidden');
     const whyBox = document.querySelector('#why-box');
-    whyBox.removeAttribute('hidden');
-    whyBox.className = 'addon-box';
+    whyBox.classList.remove('hidden');
     const whyButton = document.querySelector('#why-button');
-    whyButton.className = 'grayed';
+    whyButton.classList.add('grayed');
   }
 
   closeWhyBox() {
     this.hideWhyBox();
-    const lastRec = document.querySelector('.hiddenRec');
-    lastRec.className = 'addon-box';
-    lastRec.removeAttribute('hidden');
+    const lastRec = document.querySelector('.addon-box:last-child');
```

Comment:
I mean that there's no need to manually hide the last addon box with a hidden class, you can simply make the why box visible and style it such that it overlays the last addon box. So showing or hiding the why box wouldn't require modifying the last addon in the list at all.

Visually it would look exactly the same.